### PR TITLE
[CELADON] WW13 release notes udpated

### PR DESCRIPTION
**Pull request by swaroopbalan**
Tracked-On: None
Signed-off-by: sgnanase <sundar.gnanasekaran@intel.com>

### CODE REVIEW

File: source/release_notes.rst [hunk 1]
```diff
@@ -4,6 +4,91 @@ Release Notes
 #############
 
 .. contents:: :local:
+---------------
```
Comment (ttzeng) [inline source/release_notes.rst]: @swaroopbalan your have to add one blank line after previous directive.

File: source/release_notes.rst [hunk 2]
```diff
@@ -4,6 +4,91 @@ Release Notes
...
+
+* UI Flicker observed in few Screens for example - Under AOSP settings .
+* Lets Drive screen seen, on click of applications under "New User"
+* webM format video hangs during switch from full screen to normal : only on chrome browser
+* media file copying from external memory to internal memory fails when huge file is copied
+* saved system time changes to default after reboot
+* Call,Alarm volume and alarm sound not audible via USB headset
+* No applications seen on the main page of play store, due to google server update 
+
+--------
```
Comment (ttzeng) [inline source/release_notes.rst]: @swaroopbalan the separator is redundant here.

File: source/release_notes.rst (hunk 1 above)
Comment (swaroopbalan) [reply source/release_notes.rst]: done

File: source/release_notes.rst (hunk 2 above)
Comment (swaroopbalan) [reply source/release_notes.rst]: done

**Pull request merged**